apps/codeforafrica/src/payload/globals/Publication/PostsTab.js

```diff
@@ -0,0 +1,82 @@
+const PostsTab = {
+  label: "Posts",
+  fields: [
+    {
+      name: "stories",
+      label: "Stories",
+      type: "group",
+      fields: [
+        {
+          name: "showRecent",
+          label: "Show Recent Stories",
+          type: "checkbox",
+          required: true,
+          defaultValue: false,
+          admin: {
+            description: "Show recent stories on the individual post page",
+          },
+        },
+        {
+          name: "title",
+          label: "Title",
+          type: "text",
+          required: true,
+          defaultValue: "Recent Stories",
+          admin: {
+            condition: (_, data) => data.showRecent,
+          },
+        },
+        {
+          name: "primaryTag",
+          label: "Primary Tag",
+          type: "text",
+          required: true,
+          defaultValue: "stories",
+          admin: {
+            hidden: true,
+          },
+        },
+      ],
+    },
```

Comment:
1. UI needs to look like the rest of the globals
2. localisation must be enabled at top levels similar to other globals
```suggestion
    {
      type: "collapsible",
      label: "Story",
      fields: [
        {
          name: "recent-stories",
          label: "Recent stories",
          type: "group",
          localized: true,
          fields: [
            {
              name: "enabled",
              label: "Show recent stories",
              type: "checkbox",
              required: true,
              defaultValue: false,
            },
            {
              name: "title",
              label: "Title",
              type: "text",
              required: true,
              defaultValue: "Recent Stories",
              admin: {
                condition: (_, data) => data.showRecent,
              },
            },
            {
              name: "primaryTag",
              label: "Primary Tag",
              type: "text",
              required: true,
              defaultValue: "stories",
              admin: {
                hidden: true,
              },
            },
          ],
          admin: {
            className: "group-field-nested",
          },
        },
      ],
    },
```
<img width="608" alt="Screenshot 2023-10-09 at 09 49 58" src="https://github.com/CodeForAfrica/ui/assets/1779590/d97dc3a9-ad63-4822-bca6-2262af2d0888">